jep/0000/README.adoc

```diff
@@ -0,0 +1,322 @@
+
+Change to JDK 11 in the link:https://hub.docker.com/r/jenkins/inbound-agent[inbound agent images] that do not include an explicit JDK in their tag, like:
+
+* 4.9-1
+* 4.9-1-alpine
+* 4.9-1-windowsservercore-ltsc2019
+* alpine
+* latest
+
+Change to JDK 11 in the link:https://hub.docker.com/r/jenkins/ssh-agent/[outbound (ssh) agent images] that do not include an explicit JDK in their tag, like:
+
+* 2.1.0-alpine
+* 3.0.0
+* 3.0.0-alpine
+* alpine
+* latest
+* stretch - no change, retire the image, replace with bullseye image
+
+=== Add new tags and images
+
+Additional tags will be added for those users who require a Java 8 image.
+
+**Provide additional Docker images for Java 8** in case users have critical requirements that must use Java 8.
+
+Add JDK 8 tags for the link:https://hub.docker.com/r/jenkins/jenkins[weekly controller images] as a fallback for users requiring JDK 8:
+
+* latest-jdk8
+* slim-jdk8
+* centos7-jdk8
+* alpine-jdk8
+
+Add JDK 8 tags for the link:https://hub.docker.com/r/jenkins/jenkins[long term support controller images] that do not include an explicit JDK in their tag, like:
+
+* lts-jdk8
+* lts-slim-jdk8
+* lts-centos7-jdk8
+* lts-alpine-jdk8
+
+Add JDK 8 tags to the link:https://hub.docker.com/r/jenkins/inbound-agent[inbound agent images] that do not include an explicit JDK in their tag, like:
+
+* 4.9-1-jdk8
+* 4.9-1-jdk8-alpine
+* 4.9-1-jdk8-windowsservercore-ltsc2019
+* alpine-jdk8
+* latest-jdk8
+
+Add JDK 8 tags to the link:https://hub.docker.com/r/jenkins/ssh-agent/[outbound (ssh) agent images] that do not include an explicit JDK in their tag, like:
+
+* 2.1.0-alpine-jdk8
+* 3.0.0-jdk8
+* 3.0.0-alpine-jdk8
+* alpine-jdk8
+* latest-jdk8
+* stretch no change, retire the image, replace with bullseye image
+
+Some of the existing tags are for operating systems that have ended their standard support life.
+Add new operating system tags to use actively maintained operating systems.
+
+**Add Docker images for Debian 11 (bullseye)** in those images that include a link:https://www.debian.org/releases/stretch/[Debian 9 (stretch)] image.
+Debian 9 switched to link:https://wiki.debian.org/LTS[long term support] July 9, 2020.
+Debian 9 link:https://wiki.debian.org/LTS[long term support] will end June 30, 2022.
+
+Add Debian Bullseye image to the outbound (ssh) agent images to replace Debian stretch:
+
+* bullseye
+
+=== Stop publishing obsolete tags
+
+**Stop publishing `centos` tags** because they use CentOS 8 as a baseline and CentOS 8 is no longer receiving updates from the Red Hat Enterprise Linux upstream.
+See the link:https://blog.centos.org/2020/12/future-is-centos-stream/[CentOS blog post] that describes the change from CentOS as downstream from Red Hat Enterprise Linux to being upstream of Red Hat Enterprise Linux.
+If we want to retain a centos image, then we need to migrate from CentOS 8 to CentOS Stream 8 if an official Docker image is ever created for CentOS Stream 8.
```

Comment:
We could mention almalinux and ubi images as a drop-in replacement

Reply:
Excellent suggestion.  Added in 78c0bf92e41b50bb7a8e40b3651863fee728bd0b